src/main/java/com/datadog/api/v2/client/api/AuditApi.java

```diff
@@ -183,6 +187,70 @@ public CompletableFuture<AuditLogsEventsResponse> listAuditLogsAsync(
             });
   }
 
+  public PaginationIterable<AuditLogsEvent> listAuditLogsWithPagination() throws ApiException {
```

Comment:
I think we can simply implement this one by calling the other method with an empty parameters instance?

Reply:
Unfortunately not since there are cases where endpoint does not take optionalParams. For example: https://github.com/DataDog/datadog-api-client-java/blob/4ce582ec0f8d5a74285b2588aefd0273f144b0c4/src/main/java/com/datadog/api/v2/client/api/RumApi.java#L539

I can probably extract some of the pieces into a macro and re-use it that way the jinja2 is at least a little easier to read although the generated client will have same amount of code.